test_suite/error.slice_out_of_bounds.sugar.jsonnet

```diff
@@ -0,0 +1,3 @@
+local arr = std.range(0, 5);
+
+arr[0:10]
```

Comment:
We should exactly replicate the semantics of Python

>>> range(0, 5)[0:10]
[0, 1, 2, 3, 4]

Also supporting negative numbers.  This can all be implemented in std.slice() :)